### ISSUE TITLE
Illegal access to protected component

### ISSUE THREAD

**Issue by casella** [AMIT-FHBielefeld/PNlib#73]
The OMC NF fails when compiling [PNlib.Examples.Models.FiliP.ModellStationen.Station1](https://libraries.openmodelica.org/branches/master/PNlib/files/PNlib_PNlib.Examples.Models.FiliP.ModellStationen.Station1.err), with error
```
[PNlib 2.2/Examples/Models/FiliP/Station.mo:33:3-34:65:writable]
Error: Illegal access of protected element placeOut.
[PNlib 2.2/Examples/Models/FiliP/ModellStationen/Station1.mo:13:3-13:60:writable]
Error: Variable station.placeOut[1] not found in scope Station1.
```
Indeed, [this connect statement](https://github.com/AMIT-FHBielefeld/PNlib/blob/ab9b6b8527c0c78140365e7b105ae469d4954a64/PNlib/Examples/Models/FiliP/ModellStationen/Station1.mo#L13) refers to a [protected connector](https://github.com/AMIT-FHBielefeld/PNlib/blob/ab9b6b8527c0c78140365e7b105ae469d4954a64/PNlib/Examples/Models/FiliP/Station.mo#L33).

Could you move that connector to the public section?

**Comment by lochel** [comment]
I don't think it is that easy to fix. A few days ago, I proposed to remove the experiment annotation as a workaround until the model is fixed.

**Comment by casella** [comment]
> I don't think it is that easy to fix. A few days ago, I proposed #71 which removes the experiment annotation as a workaround until the model is fixed.

LGTM, please go ahead with that

It would be nice if we have a growing number of libraries that are supported 100% by OpenModelica. We already have some, e.g. [PlanarMechanics](https://libraries.openmodelica.org/branches/master/PlanarMechanics/PlanarMechanics.html), [PowerGrids](https://libraries.openmodelica.org/branches/master/PowerGrids/PowerGrids.html), [MEV](https://libraries.openmodelica.org/branches/master/MEV/MEV.html). PNLib is [almost there](https://libraries.openmodelica.org/branches/master/PNlib/PNlib.html), except for this issue and for [#6400](https://trac.openmodelica.org/OpenModelica/ticket/6400).

Having models in those libraries which are broken in there because of non-compliant code is certainly not a good idea.

**Comment by lochel** [comment]
Yes, I agree. I also opened #72 some days ago, since I am not sure if it should be addressed in the library.

PNlib was supported 100% by OpenModelica earlier. Would be nice to get there again!

**Comment by phannebohm** [comment]
This seems to be outdated. PNlib Examples work 100% and the model mentioned above does not exist anymore.
